.holo/README.md

```diff
@@ -10,13 +10,9 @@ hologit allows:
 3. Applying transformations to files as part of #1
    - These transformations are called "lenses"
 
-In this repository, we declare one holobranch named [release-candidate](../branches/release-candidate).
-By projecting this holobranch in GitHub Actions, individual "candidate" branches end up containing
-only the code relevant to infra/Kubernetes as well as Kubernetes code from the upstream [cluster-template](https://github.com/JarvusInnovations/cluster-template)
-repository. Then, a PR from a `candidate/<some-branch>` to `releases/<env>` (such as `releases/test`) will only show changes/content
-relevant to infra in addition to `releases/*` branches only ever containing infra code. For example:
+In this repository, we declare one holobranch named [kubernetes-workspace](../branches/kubernetes-workspace).
+By projecting this holobranch in GitHub Actions, a tree containing only the code relevant to infra/Kubernetes
+as well as Kubernetes code from the upstream [cluster-template](https://github.com/JarvusInnovations/cluster-template)
```

Comment:
Yeah I'm not sure about the best strategy there. This is a repository we actively maintain for many public/community/client consumers. Cal-ITP is pinned to a specific tagged version and it's a public open repo so it's ensured to stay static until someone bumps to a newer version. Eventually dependencies will need to be updated and at that point whoever is maintaining the infra can either fork the template or bump up to a new version. Most likely Jarvus will have already tested and documented the upgrade path to newer dependency mixes.

While there is a goal to minimize Jarvus dependencies, the project is depending on a lot of random public/community GitHub repositories for lots of things like GitHub actions and helm charts and it would be an anti-pattern to go through and fork every dependency. In cases where a Jarvus repository is linked that's just a fork of something else or a single-use repository, I'd agree with prioritizing eliminating that external dependency, but in this case this is a first-class maintained community project so I'm not sure it makes sense as a goal to just fork it into ITP preemptively

Reply:
@SorenSpicknall That said it would be a good issue to search for "Jarvus" throughout the codebase and make sure if any forks of community repos are being linked against, we're doing work to get any needed patches moved upstream so we can move the link back to targeting the upstream directly